onebusaway-android/src/main/java/org/onebusaway/android/util/ExternalIntents.kt

```
@@ -49,6 +50,78 @@ object ExternalIntents {
+        val launch = if (preferred != null) {
+            Intent(open).setPackage(preferred)
+        } else {
+            val explicit = browsers.map { Intent(open).setPackage(it) }
+            Intent.createChooser(explicit.first(), null).apply {
+                putExtra(Intent.EXTRA_INITIAL_INTENTS, explicit.drop(1).toTypedArray())
+            }
+        }
+        return try {
+            context.startActivity(launch)
+            true
+        } catch (e: ActivityNotFoundException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)